pkg/question/question.go

```diff
@@ -290,16 +294,19 @@ func AskLaunchTemplateVersion(h *ec2helper.EC2Helper, launchTemplateId string, d
 		}
 	}
 
-	optionsText := table.BuildTable(data, []string{"Option(Version Number)", "Description"})
 	question := "Launch Template Version"
+	headers := []string{"Version Number", "Description"}
 
-	answer := AskQuestion(&AskQuestionInput{
-		QuestionString: question,
-		DefaultOption:  &defaultOption,
-		OptionsString:  &optionsText,
+	model := &questionModel.SingleSelectList{}
+	err = questionModel.AskQuestion(model, &questionModel.QuestionInput{
```

Comment:
Can you add handling for non-nil values of `err`, like the other two questions have?